### PR TITLE
Score calculation to master merge time

### PR DESCRIPTION
**Pull request by AsherWright**

### CODE REVIEW

File: player scores/calculate_scores.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        player_id = row["player_id"]
+
+        cursor.execute("INSERT INTO `d2matchb_bball`.`scores` (`player_id`, `season`, `score`)" +\
+                       " VALUES (" + str(player_id) + "," + str(SEASON) + "," + str(score) + ");")
+
+    connection.commit()
+    # print all the first cell of all the rows
+    connection.close()
+
+if __name__ == '__main__':
```
Comment (WPoole) [inline player scores/calculate_scores.py:51]: Do you mean '__main_frame__'?

Comment (AsherWright) [reply player scores/calculate_scores.py:51]: @WPoole this has been fixed. please look again

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+config.json
```
Comment (AsherWright) [inline .gitignore:1]: new line please

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```
Comment (AsherWright) [inline .vscode/settings.json:3]: new line please

File: template_config.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// This is a template for our config.json file. Ask Ege for credentials.
+{
+    "host": "",
+    "user": "",
+    "pwd": "",
+    "db": ""
+}
```
Comment (AsherWright) [inline template_config.json]: new line please

**Review by AsherWright — COMMENTED** [review]
Looks ok.

File: template_config.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+//This file is a "template". You don't actually use it. It's useless. Add another file
+//in this directory called "config.json" and put what's below. That may seem stupid,
```
Comment (WPoole) [inline template_config.json:2]: No its not. Can discuss if clarification is needed.

File: template_config.json (hunk 4 above)
Comment (WPoole) [reply template_config.json]: Looks great

**Review by WPoole — APPROVED** [review]

**Pull request merged**